perf/run-benchmarks

```diff
@@ -390,6 +398,9 @@ def main() -> None:
     parser.add_argument(
         "--upload", help="upload results to semgrep dashboard", action="store_true"
     )
+    parser.add_argument(
+        "--std_only", help="only run the default semgrep", action="store_true"
```

Comment:
I'd like to change option flags to using only dashes (`--std-only`, `--plot-benchmarks`), as it is most common and expected by users.

Reply:
Will do! Our options flags are somewhat disorganized; I'm thinking about separating the corpuses into small (< 10 s), medium (< 2 min), and large (rest), and having corresponding flags

Reply:
I recently read that the fact that we use -- for flags is an accident; originally short was supposed to use -x and long argument +xxx :)